lib/postgrex/error.ex

```diff
@@ -20,4 +20,14 @@ defmodule Postgrex.Error do
   def exception(arg) do
     super(arg)
   end
+
+  def exception(:ssl, message, reason) do
+    reason = :ssl.format_error(reason)
+    %Postgrex.Error{message: "ssl #{message}: #{reason}"}
+  end
+
+  def exception(:tcp, message, reason) do
+    reason = :inet.format_error(reason)
+    %Postgrex.Error{message: "tcp #{message}: #{reason}"}
+  end
```

Comment:
> It would also be useful to still include the actual error code/atom. It is useful when searching

@gjaldon you missed this comment. or did I misunderstand?